package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nebula.js/sn-action-button",
-  "version": "1.14.0",
+  "version": "1.14.1",
```

Comment:
Not sure how you updated this, but only changing this number will not do the trick

Reply:
Is there documentation somewhere on how to do it?